examples/bri-3/src/bri/transactions/agents/transactions.agent.ts

```diff
@@ -177,6 +177,7 @@ export class TransactionAgent {
   ): Promise<TransactionResult> {
     const txResult = new TransactionResult();
 
+    // TODO: Have the hash algh 'sha256' in a single place in env instead of spread around in code
```

Comment:
agree! open issue yet?

Reply:
@Therecanbeonlyone1969  This code was copied from Ognjen's PR. This question has been addressed in PR #732